spec/integration/api_resource_comments_controller_spec.rb

```diff
@@ -64,4 +64,71 @@
       end
     end
   end
+
+  describe 'Resources Comments API' do
+    path '/api/resources/{resource_id}/resource_comments' do
+      post 'Creates a new comment' do
+        tags 'Resource comment'
+        consumes 'application/json'
+        parameter name: :resource_id, in: :path, type: :string
+        parameter name: :json_body, in: :body, schema: {
```

Comment:
I think that :json_body as parameter name is too generic. Maybe :comment ?

Reply:
the parameter described is the whole json body. Comment would be less clear (why the comment just has a content?)

Reply:
Following the example of rswag: https://github.com/rswag/rswag.  The essence of comment is the content.  
However, I don't have strong opinion in the issue so json_body is good for me.

Reply:
Following the Rswag example, I agree with Cris to name the parameter as `:comment`.  I think `:json_body` is too general.

Reply:
thanks! the example convinced me haha. It's done